src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
Well, when ghc has two different type class with same names and "similar" arguments does a weird thing of just appending names to them. Following ghc given the symbol `X` I want to create the symbol `X0` and `X1` (without any separator).  I will define an intSymbolBare inside `Names` is this is any better.